src/styles/main.scss

```
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./reset.scss";
+@import "./utility/_variables";
+@import "./utility/_mix";
+@import "./utility/_extends";
```

Personally would load vars/utils first than every other item. Might happen we'll use vars on normalize/reset styles.
```suggestion
@import "./utility/_variables";
@import "./utility/_mix";
@import "./utility/_extends";

@import "./reset.scss";
```